solution.rb

```diff
@@ -0,0 +1,39 @@
+TEXT = File.read('speech.txt')
+
+
+class WordCounter
+  @@WORDS_REGEX = /\w+[a-z']*/
```

Comment:
Wow, had no idea that a child's class variable OVERWRITES a class variable of the same name within its superclass. That's pretty surprising and now I see why class vars are not favored